sdk/lib/arbiter/arbiter.h

```diff
@@ -0,0 +1,255 @@
+#ifndef _ARBITER_H
+#define _ARBITER_H
+
+#ifndef _NTOSKRNL_
+#include <ntifs.h>
+#include <wdm.h>
```

Comment:
This is not necessary, `ntifs.h` already includes `ntddk.h`, which includes `wdm.h`.